doc/man1/openssl-cms.pod

```diff
@@ -628,10 +627,10 @@ the signers certificates.
 
 =head1 COMPATIBILITY WITH PKCS#7 FORMAT
 
-The B<smime> utility can only process the older B<PKCS#7> format. The B<cms>
-utility supports Cryptographic Message Syntax format. Use of some features
-will result in messages which cannot be processed by applications which only
-support the older format. These are detailed below.
+The L<openssl-smime(1)> utility can only process the older B<PKCS#7> format.
+The B<openssl cms> utility supports Cryptographic Message Syntax format.
```

Comment:
I actually shortened it, so now the two lines are:

```
L<openssl-smime(1)> can only process the older B<PKCS#7> format.
B<openssl cms> supports Cryptographic Message Syntax format.
```

This is one spot where I plan on keeping the explicit mention of **openssl cms**, since it's there as a direct comparison with **openssl smime**